contrib.go

```diff
@@ -0,0 +1,27 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package contrib contains common values used across all
+// instrumentation, exporter, and detector contributions
+package contrib // import "go.opentelemetry.io/contrib"
+
+// Version is the current release version of OpenTelemetry Contrib in use.
+func Version() string {
+	return "0.10.1"
```

Comment:
Maybe add a comment that you're updating this via `sed` below. 😁

Reply:
Done in #225